### PR TITLE
enrichment limits string to 1000 characters

### PR DESCRIPTION
**Pull request by AudreyAltman**
This enrichment truncates strings that are more than 1000 characters.

### CODE REVIEW

File: lib/krikri/enrichments/limit_characters.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module Krikri::Enrichments
+  ##
+  # Enrichment to limit the number of characters in a string to 1000
+  #
```
Comment (no-reply) [inline lib/krikri/enrichments/limit_characters.rb]: I know most of the enrichments don't do this, but I've been trying to get better at using YARD...
...maybe an `@example` tag, here?

File: spec/lib/krikri/enrichments/limit_characters_spec.rb [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+require 'spec_helper'
+
+describe Krikri::Enrichments::LimitCharacters do
+  it_behaves_like 'a field enrichment'
+
+  values = [{ :string => 'truncates a string over 100 characters',
```
Comment (no-reply) [inline spec/lib/krikri/enrichments/limit_characters_spec.rb]: minor typo: "100" for "1000".

File: spec/lib/krikri/enrichments/limit_characters_spec.rb [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+              'in urna maximus dictum. Phasellus tempor tincidunt lectus '\
+              'aliquam eleifend. Morbi in commodo massa, sit amet bibendum '\
+              'quam. Sed efficitur velit libero, nec pulvinar diam tempor a. '\
+              'In eu ullamcorper risus, non pretium felis. Nam rutrum augue '\
+              'vel quam lacinia, non feugiat nunc fermentum. Aliquam lacinia '\
+              'rutrum lorem sed commodo. Nam tempor vel nisl eget vestibulum. '\
+              'Nunc lorem velit, euismod a ipsum non, consequat vulputate '\
+              'tellus. Phasellus sapien ipsum...'
+            },
+            { :string => 'leaves strings under 100 characters alone',
```
Comment (no-reply) [inline spec/lib/krikri/enrichments/limit_characters_spec.rb]: same typo as above.

**Comment by no-reply** [comment]
:rocket: :fireworks:

**Comment by no-reply** [comment]
Looks good! Unless @anarchivist wants to take a peek, I say :squash:

**Comment by AudreyAltman** [comment]
:squash:ed

**Pull request merged**